dotnet/CoreLib/DataFormats/Image/AzureFormRecognizer/AzureFormRecognizerEngine.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure;
+using Azure.AI.FormRecognizer;
+using Azure.AI.FormRecognizer.Models;
+
+namespace Microsoft.SemanticMemory.DataFormats.Image.AzureFormRecognizer;
+
+/// <summary>
+/// Wrapper for the Azure.AI.FormRecognizer.
+/// </summary>
+public class AzureFormRecognizerEngine : IOcrEngine
+{
+    private readonly FormRecognizerClient formRecognizerClient;
+
+    /// <summary>
+    /// Creates a new instance of the AzureFormRecognizerOcrEngine passing in the Form Recognizer endpoint and key.
+    /// </summary>
+    /// <param name="endpoint">The endpoint for accessing a provisioned Azure Form Recognizer instance</param>
+    /// <param name="credential">The AzureKeyCredential containing the provisioned Azure Form Recognizer access key</param>
+    public AzureFormRecognizerEngine(string endpoint, AzureKeyCredential credential)
+    {
+        this.formRecognizerClient = new FormRecognizerClient(new Uri(endpoint), credential);
```

Comment:
Looks like we have a new service version that we can use: https://learn.microsoft.com/en-us/dotnet/api/azure.ai.formrecognizer.documentanalysis.documentanalysisclient?view=azure-dotnet

Reply:
Updated with new client - its a lot more flexible...for parity, we use: `prebuilt-read` model (plain text ocr)

Reply:
Verified Azure Forms integration